avocado_vt/test.py

```diff
@@ -276,13 +276,18 @@ def setUp(self):
                         logging.info("archiving libvirtd debug logs")
                         from virttest import utils_package
                         if utils_package.package_install("tar"):
-                            archive = os.path.join(libvirtd_log.strip(os.path.basename(libvirtd_log)),
-                                                   "libvirtd.tar.gz")
-                            cmd = ("tar -zcf %s -P %s"
-                                   % (pipes.quote(archive),
-                                      pipes.quote(libvirtd_log)))
-                            if process.system(cmd) == 0:
-                                os.remove(libvirtd_log)
+                            if os.path.isfile(libvirtd_log):
+                                archive = os.path.join(libvirtd_log.strip(
```

Comment:
In general yes, but as you're touching the code, would you please also fix the incorrect use of `strip` using a `basename`? It should be simply: `archive = os.path.join(os.path.dirname(libvirtd_log), "libvirtd.tar.gz")`

Reply:
Updated.